client/me/help/main.jsx

```diff
@@ -54,7 +63,7 @@ class Help extends React.PureComponent {
 				),
 			},
 			{
-				link: 'https://en.support.wordpress.com/start/',
+				link: `https://${ getSupportLocale() }.support.wordpress.com/start`,
```

Comment:
Nitpick: Although it will be eventually appended by a trailing slash as `xx.support.wordpress.com/start/`, because of the URL rewriting rules. We could consider to just add the trailing slash here on our own.
In my opinion this is clearer, since for with / without trailing slashes can mean different things under different server configurations.

Reply:
Thanks. I meant to leave it in. Mistake on my part. 👍